atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -854,7 +856,7 @@ private boolean isThoroughlySwept(TableReference tableRef) {
     }
 
     private List<Map.Entry<Cell, byte[]>> getPostFilteredWithLocalWrites(final TableReference tableRef,
-                                                                     final SortedMap<Cell, byte[]> postFiltered,
```

Comment:
I believe it is essential that this is a `SortedMap`. We should discuss offline

Reply:
Or, at least, that entries provided by the iterator here are in sorted order. I get that we haven't changed the concrete implementation, but would prefer to codify this guarantee in the type system.